code/game/machinery/Sleeper.dm

```diff
@@ -28,15 +28,25 @@
 /obj/machinery/sleep_console/New()
 	..()
 	spawn( 5 )
+		update_icon()
 		if(orient == "RIGHT")

```

Comment:
It's rather questionable why this is using a variable called orient instead of dir